ui/lib/components/CardTabs/index.module.less

```diff
@@ -3,7 +3,7 @@
   margin-right: -@padding-page;
 
   :global {
-    .ant-tabs-bar {
+    .ant-tabs-nav {
```

Comment:
Maybe better to find some way to remove this rule (which depends on the internal implementation of Antd's Tab).

Reply:
ok, let me try it.

Reply:
It works!